mix.exs

```diff
@@ -28,8 +28,8 @@ defmodule KinoExplorer.MixProject do
   defp deps do
     [
       {:kino, "~> 0.12.3"},
-      {:explorer, "~> 0.8.0"},
-      {:rustler, "~> 0.29.0", optional: true},
+      {:explorer, "~> 0.8.1"},
```

Comment:
Btw, you don't have to worry about updating dependencies in cases like these either, unless there is a breaking change, but no harm if you do :)